interface/grpc/core/core.go

```diff
@@ -173,7 +173,7 @@ func (s *Server) ListenResult(request *coreapi.ListenResultRequest, stream corea
 				return err
 			}
 			if err := stream.Send(&coreapi.ResultData{
-				ExecutionID:   execution.ID,
+				ExecutionHash: string(execution.Hash),
```

Comment:
Converting bytes to string will not work..
The hash is not directly convertible to string. You'll need to use base58 conversion functions.